docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  web:
+    build: .
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- af0361e8760870b166ebb1a33ff69327 -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - Files or Directories Accessible to External Parties
_Low CWE-552_
The docker service container is configured to permit writing to the root filesystem. This makes some security attack vectors such as privilege escalation, denial-of-service or authorization bypass possible since the container instance's filesystem can be tampered with.



### How to fix
Unspecified remediation.